topology/ANL/ANL ASC/ANLASC.yaml

```diff
@@ -0,0 +1,243 @@
+Disable: false
+GridType: OSG Production Resource
+GroupID: 283
+Resources:
+  ANL-ATLAS-GRIDFTP1:
+    Active: true
```

Comment:
According to Marina, resources can be `Active: false` if it's a new resource but not ready to accept jobs/data transfer/etc. `Disable` is generally for resources that were active and retired (or are in downtime maybe?). The difference seems superfluous to me. Would it be easy to count the different combinations of Active/Disable with their true/false values? Does GRACC/GGUS/AGIS differentiate between the two states?

Reply:
Maybe it works out to something like this:

| Active | Disable | Meaning |
|----------|-----------|--------------|
| true    | true    | downtime   |
| true     | false | normal operation |
| false    | true  | retired/initial setup |
| false    | false | ??? |